src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -73,9 +85,46 @@ public class DevBoxInstance : IComputeSystem
         get; private set;
     }
 
+    public IEnumerable<ComputeSystemProperty> Properties
+    {
+        get; set;
+    }
+
     public DevBoxInstance(IDevBoxAuthService devBoxAuthService)
     {
         _authService = devBoxAuthService;
+        Properties = new List<ComputeSystemProperty>();
+    }
+
+    private void ProcessProperties()
+    {
+        if (!Properties.Any())
+        {
+            var properties = new List<ComputeSystemProperty>();
+            if (CPU is not null)
+            {
+                var cpu = new ComputeSystemProperty(int.Parse(CPU, CultureInfo.CurrentCulture), ComputeSystemPropertyKind.CpuCount);
+                properties.Add(cpu);
+            }
+
+            if (Memory is not null)
+            {
+                int memoryInGB = int.Parse(Memory, CultureInfo.CurrentCulture);
+                long memoryInBytes = memoryInGB * 1073741824L;

```

Comment:
Can probably make the gigabyte value a constant called OneGbInBytes for example and then reuse it for the disk size

Reply:
Resolved.